### PR TITLE
Add feature: database service now fetches event data from Google API

### PR DESCRIPTION
**Pull request by ajtran303**
Major Changes

  - We have a new script! `python -m db.service`
    - When invoked, it will fetch up-to-date data from Google Calendars
    - A Heroku Scheduler will run it 3 times a day:
      - 12:00AM
      - 8:30AM
      - 12:30PM

  Minor Changes

  - Removed python-dotenv, as it was not being used
  - Deleted `data/calendars.py` as it was obsolete

Co-authored-by: Michael Evans <62968371+michaeljevans@users.noreply.github.com>
Co-authored-by: Ashkan Abbasi <47463190+Ashkanthegreat@users.noreply.github.com>

### CODE REVIEW

**Comment by ajtran303** [comment]
closes #8

File: db/service.py [hunk 1]
```diff
@@ -0,0 +1,130 @@
+from db.calendars import db
+from googleapiclient import discovery
+from oauth2client import client
+import httplib2
+import json
+import datetime
+
+saved_credentials = list(db.tokens.find({}))[0] # SMC
```
Comment (ajtran303) [inline db/service.py:8]: First, we use our saved credentials from the database

File: db/service.py [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+del saved_credentials['_id']
+saved_credentials = json.dumps(saved_credentials)
+
+refreshed_credentials = client.OAuth2Credentials.from_json(saved_credentials) # SMD
```
Comment (ajtran303) [inline db/service.py:12]: Then, we make sure that we have refreshed credentials

File: db/service.py [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+
+db.tokens.drop()
```
Comment (ajtran303) [inline db/service.py:14]: Delete the old credentials

File: db/service.py [hunk 4]
```diff
@@ -0,0 +1,130 @@
...
+db.tokens.insert_one(json.loads(refreshed_credentials.to_json()))
```
Comment (ajtran303) [inline db/service.py:15]: Add the refreshed credentials to replace the old ones

File: db/service.py [hunk 5]
```diff
@@ -0,0 +1,130 @@
...
+        except KeyError:
+            location = None
+
+        doc = { "start": start,
+                "end": end_time,
+                "location": location,
+                "summary": summary,
+                "htmlLink": htmlLink }
+
+        events_doc[calendar_doc].append(doc)
```
Comment (ajtran303) [inline db/service.py:89]: This makes each `doc` go into the correct `events_doc` array.

File: db/service.py [hunk 6]
```diff
@@ -0,0 +1,130 @@
...
+
```
Comment (ajtran303) [inline db/service.py:90]: The rest of this file is similar to the `db.seed` utility

**Review by michaeljevans — APPROVED** [review]
Fetching from the Google Calendar API is working!

**Review by ajtran303 — COMMENTED** [review]
Hopefully this works in production!

**Pull request merged**